dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/jaxrs/impl/UriBuilderImpl.java

```diff
@@ -853,8 +853,8 @@ private String buildParams(MultivaluedMap<String, String> map, char separator) {
                     } else {
                         val = URITemplate.createExactTemplate(val).encodeLiteralCharacters(isQuery);
                     }
+                    b.append('=');
```

Comment:
I think that this is the only difference between our code and the CXF code, so we should be able to delete this source file now. (we also have `@FFDCIgnore`s, etc. but those aren't necessary when pulling in source from CXF).